index.html

```diff
@@ -5,12 +5,41 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Hello microverse</title>
-    <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" media="screen and (min-width: 600px)" href="assets/css/smallscreen.css">
+    <link rel="stylesheet" media="screen and (max-width: 600px)" href="assets/css/widecreen.css">
```

Comment:
- [ ] Your style is not working, Please remove the media attribute from your linked stylesheets. Media quarries must be used in CSS files.